app/controllers/manage_collections_controller.rb

```diff
@@ -1,12 +1,12 @@
 # :nodoc:
-class ManageMetadataController < PagesController
+class ManageCollectionsController < PagesController
```

Comment:
If there are any changes from my comments to `drafts_controller.rb`, this may require a modification as well.